receiver/azureeventhubreceiver/README.md

```diff
@@ -36,6 +45,48 @@ receivers:
 
 This component can persist its state using the [storage extension].
 
+## Encoding
+
+### raw
+
+The "raw" encoding maps the AMQP properties and data into the
+attributes and body of an OpenTelemetry LogRecord, respectively.
+The body is represented as a raw byte array.
+
+### data
+
+The "data" encoding extracts the Azure log records from the AMQP
+message data, parses them, and maps the fields to OpenTelemetry
+attributes. The table below summarizes the mapping between the 
+[Azure common log format](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/resource-logs-schema)
+and the OpenTelemetry attributes.
+
+
+| Azure                            | OpenTelemetry                          | 
+|----------------------------------|----------------------------------------|
+| time (required)                  | time_unix_nano (field)                 | 
+| resourceId (required)            | azure.resource.id (attribute)          | 
```

Comment:
Should this be a resource attribute?

Reply:
Done. Although Azure shouldn't mix logs from difference resources in the same message, the implementation of this will fail if it does.